uninstaller/uninstall-helper.ps1

```diff
@@ -72,7 +72,12 @@ $pathFolders | foreach {
 [string] $delimitedFolders = $folderList -Join ';'
 [Environment]::SetEnvironmentVariable("Path", $delimitedFolders, "User")
 
-Remove-Item -Path "HKLM:\SOFTWARE\WOW6432Node\Microsoft\Windows\CurrentVersion\Uninstall\DevelopmentSuite$timeStamp"
```

Comment:
It seems that the only difference between what work and what does not is `-Recurse` parameter.

Reply:
every time when we install it generate random key in `"HKLM:\SOFTWARE\WOW6432Node\Microsoft\Windows\CurrentVersion\Uninstall\"` that is why I am matching the key name and deleting it.

Reply:
The key name formed from two parts DevelopmentSuite + $timeStamp from line [#5](https://github.com/redhat-developer-tooling/developer-platform-install/pull/1096/files#diff-61d71e285c0cfd8483e06293eda5583aL6). Why iterate when you can just have it as `"HKLM:\SOFTWARE\WOW6432Node\Microsoft\Windows\CurrentVersion\Uninstall\DevelopmentSuite$timeStamp"`?